models/op/governance/optimism/op_governance_optimism_delegates_addresses.sql

```diff
@@ -0,0 +1,136 @@
+    ,(0x0af3c72452afa3271c4cb3e4027b0a62477bfbc6, 'jay-paul.eth','jay-paul.eth')
+    ,(0x0e66cf7a7e7a7f4574670f6c2c9d463e1478a06c, 'frankshih.eth', 'frankshih.eth')
+    ,(0x9C8984a1b26Ea82fFcC7a4Ef058b303A85b7fA25,'ENS',NULL)
+    ,(0x489863b61C625a15C74FB4C21486baCb4A3937AB,'Thales',NULL)
+    ,(0x38E77E487518851f47e8470433D69F3119304D7C,'Kwenta',NULL)
+    ,(0x2f30c18491435cb8bf24047d3398167ac327f750,'Lyra',NULL)
+    ,(0xc94ae9860B37170c08e51f453F8124A86cd7c0Cd,'Paraswap',NULL)
+    ,(0x3576aafaffe4c07f894447265b391856377157db,'Connext',NULL)
+    ,(0x13DEbC7a8ef08a97531d0F9a7eaBF2B0271c5054,'Polynomial',NULL)
+    ,(0x352Fb838A3ae9b0ef2f0EBF24191AcAf4aB9EcEc,'dHEDGE',NULL)
+    ,(0xE36E5aFD431bbFd23598d2C37b018D2Bde5eB5e1,'Uniswap Foundation',NULL)
+    ,(0xA3Ebc4a674AC613F797B45B4BDc1c1dDAb13836a,'LayerZero',NULL)
+    ,(0xb074ec6c37659525EEf2Fb44478077901F878012,'Velodrome',NULL)
+    ,(0xaa31cb426e2c83cda9c850874d166d7fca836b14,'Rubicon',NULL)
+    ,(0xC70ABe97baF38782Fa479b23311b611D0e82076b,'Pika',NULL)
+    ,(0xDC01EA5b6B48187F5Aa1e422E90Dc01d03217B76,'Hop',NULL)
+    ,(0x07963001638c2e8283Ef7F3dABF4242Eaf5d4F79,'Nested',NULL)
+    ,(0x6fd2bd90d50edee139103454116f252f6f5ec928,'Socket',NULL)
+    ,(0x34b9422884969b3eeba798723429697053271c8a,'Mirror',NULL)
+    ,(0x2a185c8a3c63d7bfe63ad5d950244ffe9d0a4b60,'Beethoven/Balancer',NULL)
+    ,(0xc420C9d507D0E038BD76383AaADCAd576ed0073c,'Curve',NULL)
+    ,(0x1Dd174bF2e090d0F1b1360Ac8905780429B2642B,'Clipper',NULL)
+    ,(0x9Ff6e05e9c5f4E2f5ed8FD771C98E0c07E933b85,'Odos',NULL)
+    ,(0xe93d59cc0bcecfd4ac204827ef67c5266079e2b5,'404 DAO', NULL)
+    ,(0xd0214b7e8a7821a5cb07024bc00d64ece8cc1067,'esmeralda.eth','esmeralda.eth')
+    ,(0xd3efbc65713c307d3057b7e0979a996f6a9799ac,'Apetimism',NULL)
+    ,(0x8ccde64d05a7677d80a5960cb75fb8de486cf98d,'Boiler Blockchain',NULL)
+    ,(0x8f5415415d9200ccd8523f3ee88f96f476141cc3,'Rabbithole',NULL)
+    --
+    ,(0x11cf2f033b079fbb195425bfcb7d5dac4db2d752,'Gauntlet',NULL)
+    ,(0x3fb19771947072629c8eee7995a2ef23b72d4c8a,'PGov',NULL)
+    ,(0x8d5579eda0d6f0515c833ebd36bbaf116e11613a,'Collab.Land',NULL)
+    ,(0xc17a59dc5fc0df4853ee71e6dd7640e5e4954f0f,'Clique',NULL)
+    ,(0x5ab0fdc6f0ca9ea178c95ceccf944a4ce1aedddd,'Galxe','galxe.eth')
+    ,(0x8932eb23bad9bddb5cf81426f78279a53c6c3b71,'LiFi','lifiprotocol.eth')
+    ,(0xeb1afebd776c9669ce0eb3cb2e14346751223c60,'Kyberswap',NULL)
+    ,(0x59761d97585985a4dabeea984eefc0aa18ac02a8,'Thirdweb',NULL)
+    ,(0x460849d8139ae4b5c1b6669411832194b82a2863,'Multichain',NULL)
+    ,(0x5c070a099b0e02a6ae56883ff5b657129cbcb6e5,'Gelato',NULL)
+    ,(0x20f3880a281092dbc6699e5d5a0ff5feb3d3db1a,'Celer Network',NULL)
+    ,(0x2b7839e74e8a9145e594810508006b3e856088f8,'North Carolina A&T (AggieDAO)',NULL)
+    ,(0x5dA0b013a248E11F24c3d465693F1E3E858bA5D1,'NEU Blockchain','neublockchain.eth')
+    
+) a (address, name, ens)
+
+)
+
+SELECT address,
+name,
+ens FROM address_names
+WHERE rn = 1 --exclude dupes
```

Comment:
We can also use an `unique` test for address field.

Reply:
i agree with this comment, we should apply a unique test on the `address` column rather than force a row number function to run, especially since this is a manually built static dataset anyway. let's look to remove that function/filter, then apply test in schema file